assets/js/googlesitekit/api/cache.js

```diff
@@ -21,6 +21,13 @@
  */
 import { HOUR_IN_SECONDS } from '../../util';
 
+/**
+ * Cache prefix scoped to user session and blog_id.
+ *
+ * @since n.e.x.t
+ */
+const STORAGE_PREFIX = global._googlesitekitBaseData.storagePrefix;
```

Comment:
This is somewhat confusing to be a separate similar named value from the one below. We want to make sure this isn't accidentally used elsewhere in the module where we should really be using the full key. How about we inline it below? Do we need it elsewhere as a separate value? If not, let's inline it.